tests/components/cover/test_init.py

```diff
@@ -0,0 +1,52 @@
+"""The tests for the cover platform."""
+import asyncio
+
+from homeassistant.components.cover import (SERVICE_OPEN_COVER,
+                                            SERVICE_CLOSE_COVER)
+from homeassistant.components import intent
+import homeassistant.components as comps
+from tests.common import async_mock_service
+
+
+@asyncio.coroutine
```

Comment:
We've migrated to `async`/`await` syntax.

Reply:
Roger that.